### PR TITLE
[JP] Translation of advanced/behaviours #1293

### PR DESCRIPTION
**Pull request by gimKondo**
I added Japanese translation for lesson of advanced/behaviours.
(refer to #1293 )

### CODE REVIEW

**Comment by riseshia** [comment]
Ping @elixirschool/japanese

**Comment by riseshia** [comment]
cc @3100

File: jp/lessons/advanced/behaviours.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```elixir
+defmodule Example.Worker do
+  @callback init(state :: term) :: {:ok, new_state :: term} | {:error, reason :: term}
+  @callback perform(args :: term, state :: term) ::
+              {:ok, result :: term, new_state :: term}
+              | {:error, reason :: term, new_state :: term}
+end
+```
+
+これで `init/1` は任意の値を受け取り、 `{:ok, state}` または `{:error, reason}` というタプルを返す関数として定義されました。これは至って標準的な初期化です。 `perform/2` 関数はワーカー用の引数と初期化時のステートを受け取り、GenServersと同じように `{:ok, result, state}` または `{:error, reason, state}` を返すことが期待されます。
```
Comment (riseshia) [inline jp/lessons/advanced/behaviours.md]: (typo) `GenServers` -> `GenServer`

File: jp/lessons/advanced/behaviours.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+## ビヘイビアを利用する
+
+我々の定義したビヘイビアを使って、さまざまなモジュールに同じ公開APIを共有させられます。ビヘイビアをモジュールに追加するのは簡単で、 `@behaviour` 属性を使います。
```
Comment (riseshia) [inline jp/lessons/advanced/behaviours.md]: (maybe) `共有させられます。` -> `共有できます。`

File: jp/lessons/advanced/behaviours.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  end
+
+  defp compress({name, files}), do: :zip.create(name, files)
+
+  defp respond({:ok, path}, opts), do: {:ok, path, opts}
+  defp respond({:error, reason}, opts), do: {:error, reason, opts}
+end
+```
+
+コンパイルしてみると、次のような警告が表示されるはずです。
```
Comment (riseshia) [inline jp/lessons/advanced/behaviours.md]: (maybe) `表示されるはずです。` -> `表示されます。`

> Now when we compile our code we `should see` a warning:

**Comment by gimKondo** [comment]
@riseshia Thank you for reviewing. I accepted your opinions and fix those soon.

**Comment by riseshia** [comment]
:tada:

**Pull request merged**